diff -urN aaa/Mem/rox_run Mem/rox_run
--- aaa/Mem/rox_run	Mon Mar  4 12:49:22 2002
+++ Mem/rox_run	Tue Jul  2 15:35:11 2002
@@ -15,11 +15,11 @@
 fi
 
 if [ -z "$LIBDIRPATH" ]; then
-    LIBDIRPATH=$HOME/lib:/usr/local/lib:/usr/lib export LIBDIRPATH
+    LIBDIRPATH=$HOME/lib:/usr/local/lib:/usr/lib:/usr/X11R6/lib export LIBDIRPATH
 fi
 
 if [ -z "$APPPATH" ]; then
-    APPPATH=$HOME/Apps:/usr/local/apps export APPPATH
+    APPPATH=$HOME/Apps:/usr/local/apps:/usr/X11R6/lib/ROX-apps export APPPATH
 fi
 
 IFS=:
