diff -urN Mem.new/libdir Mem/libdir
--- Mem.new/libdir	2005-04-18 20:52:54.542132848 +0200
+++ Mem/libdir	2005-04-18 21:06:55.927222872 +0200
@@ -8,12 +8,12 @@
 # $Id$
 
 if [ x"$LIBDIRPATH" = x ]; then
-  LIBDIRPATH="$HOME/lib:/usr/local/lib:/usr/lib"
+  LIBDIRPATH="$HOME/lib:/usr/local/lib:/usr/lib:/usr/lib/rox:/usr/lib64/rox"
   export LIBDIRPATH
 fi
 
 if [ x"$APPDIRPATH" = x ]; then
-  APPDIRPATH="$HOME/Apps:/usr/local/apps:/usr/apps"
+  APPDIRPATH="$HOME/Apps:/usr/local/apps:/usr/apps:/usr/lib/rox:/usr/lib64/rox"
   export APPDIRPATH
 fi
 
